docs/production-deployment/cloud/account-setup/users.mdx

```diff
@@ -131,7 +134,9 @@ Temporal strongly recommends the following precautions when assigning the Accoun
 
 - Assign the role to at least two users in your organization.
   Otherwise, limit the number of users with this role.
-- Associate an email address with each Account Owner role so Temporal Cloud support can be in contact for urgent situations.
+- Associate an actual person’s email address to the Account Owner, not a shared or generic one, so Temporal Support can contact the right person in urgent situations.
```

Comment:
```suggestion
- Associate a person’s direct email address to the Account Owner, rather than a shared or generic address, so Temporal Support can contact the right person in urgent situations.
```